src/current/v25.4/upgrade-with-pcr.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Upgrade with Physical Cluster Replication Enabled
+summary: Upgrade your primary and standby clusters when using PCR.
+toc: true
+docs_area: manage
+---
+
+When [**physical cluster replication (PCR)**]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}) is enabled, you must use the process on this page to upgrade your clusters. This process ensures that the standby cluster safely upgrades before the primary cluster, preventing any version incompatibility. You cannot replicate data from a cluster on a newer version to a cluster on an older version. 
```

Comment:
```suggestion
To safely upgrade a cluster with [**physical cluster replication (PCR)**]({% link {{ page.version.version }}/physical-cluster-replication-overview.md %}), you must upgrade the standby cluster before the primary cluster to avoid version incompatibility. You cannot replicate data from a cluster on a newer version to a cluster on an older version. 

This page describes the process for safely upgrading a cluster that has PCR enabled.
```